demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1154,10 +1176,10 @@ export class DemoMeetingApp
         'Available Downlink Bandwidth: Unknown';
     }
 
-    this.hasChromiumWebRTC && this.isButtonOn('button-video-stats') && this.getAndShowWebRTCStats();
+    this.isButtonOn('button-video-stats') && this.showVideoWebRTCStats(this.videoUpstreamMetricReport, this.videoDownstreamMetricReport);
   }
 
-  getAndShowWebRTCStats(): void {
+  showVideoWebRTCStats(videoUpstreamMetricReport: { [id: string]: { [id: string]: {} } }, videoDownstreamMetricReport: { [id: string]: { [id: string]: {} } }): void {
```

Comment:
I would suggest merge this function with showVideoStats. All it does it to find the videoTile.